neard/Cargo.toml

```diff
@@ -69,15 +69,15 @@ expensive_tests = ["nearcore/expensive_tests"]
 no_cache = ["nearcore/no_cache"]
 rosetta_rpc = ["nearcore/rosetta_rpc"]
 json_rpc = ["nearcore/json_rpc"]
-protocol_feature_fix_staking_threshold = ["nearcore/protocol_feature_fix_staking_threshold"]
-protocol_feature_simple_nightshade_v2 = ["nearcore/protocol_feature_simple_nightshade_v2"]
+protocol_feature_fix_staking_threshold = [
```

Comment:
Why do we have the formatting changes in all the cargo files? I've seen this quite often... Don't we have some standard?

Reply:
Yeah apparently different developer have different or none formatters and this happens. I'm disabling my formatter and I'm not going to give it another thought. If you feel ambitious we can setup something in CI to enforce some common formatting.